changelogs/fragments/migrate_ec2_instance.yml

```diff
@@ -0,0 +1,4 @@
+major_changes:
+  - ec2_instance - The module has been migrated from the ``community.aws`` collection. Playbooks using the Fully Qualified Collection Name for this module should be updated to use ``amazon.aws.ec2_instance``.
+  - ec2_instance_info - The module has been migrated from the ``community.aws`` collection. Playbooks using the Fully Qualified Collection Name for this module should be updated to use ``amazon.aws.ec2_instance_info``.
+~                                                                                                                                                                                                                      
```

Comment:
```suggestion
```

Reply:
I did an amend on the requested changes, since we're not going to squash.  I think if it were more than 4 lines I'd probably reconsider that?  But for this it should be ok, IMO.